src/types.js

```diff
@@ -12,3 +13,39 @@ export const link = PropTypes.shape({
 export const pageContext = PropTypes.shape({
   fileRelativePath: PropTypes.string,
 });
+
+export const quickstartDashboard = PropTypes.shape({
+  name: PropTypes.string.isRequired,
+  url: PropTypes.string.isRequired,
+  description: PropTypes.string,
+  screenshots: PropTypes.arrayOf(PropTypes.string),
+});
+
+export const quickstartAlert = PropTypes.shape({
+  name: PropTypes.string.isRequired,
+  url: PropTypes.string.isRequired,
+  type: PropTypes.oneOf(['BASELINE', 'OUTLIER', 'STATIC']).isRequired,
```

Comment:
We could break these types out into an enum, similar to the support level. Not required at all, just an idea.

Reply:
Updated!